cmd/enduro-a3m-worker/main.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/spf13/pflag"
 	"go.artefactual.dev/tools/log"
+	temporal_tools "go.artefactual.dev/tools/temporal"
```

Comment:
If you like, you can enforce this alias here: https://github.com/artefactual-sdps/enduro/blob/8aa0da8a1c1afd7cceaabfeac1abdbfd7393f9f8/.golangci.yml#L41-L47

Reply:
I don't like to enforce it.